var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -344,6 +344,9 @@ def libs(self):
     # See https://github.com/spack/spack/issues/28273
     patch("pthread-stack-min-fix.patch", when="@1.69.0:1.72.0")
 
+    # https://www.intel.com/content/www/us/en/developer/articles/technical/building-boost-with-oneapi.html
+    patch("1.78-intel-linux-jam.patch", when="@1.78 %oneapi")
```

Comment:
Is this problem for 1.79?

Reply:
This is a 1.78 specific issue. Without this patch you get the error described here: https://github.com/spack/spack/issues/29352